miner/src/block_assembler.rs

```diff
@@ -538,7 +538,7 @@ mod tests {
     use ckb_verification::{BlockVerifier, HeaderResolverWrapper, HeaderVerifier, Verifier};
     use jsonrpc_types::{BlockTemplate, CellbaseTemplate};
     use numext_fixed_hash::H256;
-    use std::convert::TryInto;
+    use std::convert::Into;
```

Comment:
Unless we use `no_std`, this is unnecessary. (not a request change, just a comment)